test-cases/tools/lib/test-case.ts

```diff
@@ -111,14 +117,24 @@ function extractCategory(file: string): string {
     return path.basename(path.dirname(file));
 }
 
-function loadTestCases(testDirectory?: string): TestCase[] {
-    return walk(testDirectory || TEST_DIR, TEST_FILTER).map(loadTestCase);
+function loadTestCases(
+    productName?: string,
```

Comment:
The productName can not be optional here and mandatory in the loadTestCase method
```suggestion
    productName: string,
```

Reply:
@b1zzu I was thinking to make `productName` parameter optional for both `loadTestCases` and `loadTestCase` methods. I.e. if `productName` is not specified, return all test cases (for all products). 

This would allow to not filter out test cases based on the product for `tools.sh export` command, as you suggested above.

wdyt?

Reply:
yes and no, the problem is that if the `test.enviroments` and `test.targets` will be undefined if the product is not specified, also right now I've changed again opinion, and proabbly keeping the mandatory product filter for the `export` cmd is better :sweat_smile: